api/src/main/resources/org/openmrs/api/db/hibernate/PatientProgramAttribute.hbm.xml

```diff
@@ -0,0 +1,55 @@
+<?xml version="1.0"?>
+<!--
+
+    This Source Code Form is subject to the terms of the Mozilla Public License,
+    v. 2.0. If a copy of the MPL was not distributed with this file, You can
+    obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+    the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+
+    Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+    graphic logo is a trademark of OpenMRS Inc.
+
+-->
+<!DOCTYPE hibernate-mapping PUBLIC
+        "-//Hibernate/Hibernate Mapping DTD 3.0//EN"
+        "http://www.hibernate.org/dtd/hibernate-mapping-3.0.dtd">
+
+<hibernate-mapping package="org.openmrs">
+
+    <class name="org.openmrs.PatientProgramAttribute" table="patient_program_attribute">
+
+        <id name="patientProgramAttributeId" type="int" column="patient_program_attribute_id">
+            <generator class="native">
+                <param name="sequence">patient_program_attribute_id_seq</param>
+            </generator>
+        </id>
+
+        <many-to-one name="patientProgram" class="org.openmrs.PatientProgram" not-null="true" column="patient_program_id"/>
```

Comment:
No need to include the package on the class name, since we already have it above.